actions/crib-deploy-environment/action.yml

```diff
@@ -112,11 +112,36 @@ runs:
         enable-aws: true
         aws-region: ${{ inputs.aws-region }}
 
-    - name: Generate CRIB Namespace Name
+    - name: Generate CRIB Namespace Name and labels
       id: generate-ns-name
       shell: bash
       run: |
         echo "devspace-namespace=crib-ci-$(uuidgen | cut -c1-5)" >> $GITHUB_OUTPUT
+        # This will work only for PRs, push event itself does not contain PR information.
+        echo "pr-number=${{ github.event.number }}" >> $GITHUB_OUTPUT
+        echo "workflow-job=${{ github.job }}" >> $GITHUB_OUTPUT
+        echo "repo-name=$(basename $GITHUB_REPOSITORY)" >> $GITHUB_OUTPUT
+        echo "branch-name=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
+        echo "commit-sha=${{ github.sha }}" >> $GITHUB_OUTPUT
+        echo "workflow-run-number=${{ github.run_number }}" >> $GITHUB_OUTPUT
+
+    - name: Crate and label CRIB namespace
+      shell: bash
+      run: |
+        NAMESPACE="${{ steps.generate-ns-name.outputs.devspace-namespace }}"
+        echo "Creating $NAMESPACE Kubernetes namespace.."
+        kubectl create ns "$NAMESPACE"
```

Comment:
Doesn't devspace create the namespace already? I might be misremembering.

Reply:
It does, but if you remember it starts deploying immediately and fails because Kyverno needs some time to add the role binding.